kubernetes-client-openapi-common/src/main/java/io/micronaut/kubernetes/client/openapi/credential/ReactiveKubernetesTokenLoader.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2017-2025 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.kubernetes.client.openapi.credential;
+
+import io.micronaut.core.annotation.NonNull;
+import org.reactivestreams.Publisher;
+
+/**
+ * The loader for bearer token used in kubernetes api service authentication.
+ */
+public interface ReactiveKubernetesTokenLoader extends TokenLoader {
+
+    /**
+     * Gets a bearer token for request authentication.
+     *
+     * @return bearer token
+     */
+    @NonNull Publisher<String> getToken();
```

Comment:
Annotate with [`@SingleResult`](https://docs.micronaut.io/latest/api/io/micronaut/core/async/annotation/SingleResult.html): 

```suggestion
    @SingleResult
    @NonNull Publisher<String> getToken();
```

Reply:
Done